schema-sandbox/schema.json

```diff
@@ -64,6 +126,8750 @@
+                            "wng",
+                            "wni",
+                            "wnk",
+                            "wnm",
+                            "wnn",
+                            "wno",
+                            "wnp",
+                            "wnu",
+                            "wnw",
+                            "wny",
+                            "woa",
+                            "wob",
+                            "woc",
+                            "wod",
+                            "woe",
+                            "wof",
+                            "wog",
+                            "woi",
+                            "wok",
+                            "wol",
+                            "wom",
+                            "won",
+                            "woo",
+                            "wor",
+                            "wos",
+                            "wow",
+                            "woy",
+                            "wpc",
+                            "wra",
+                            "wrb",
+                            "wrd",
+                            "wrg",
+                            "wrh",
+                            "wri",
+                            "wrk",
+                            "wrl",
+                            "wrm",
+                            "wrn",
+                            "wro",
+                            "wrp",
+                            "wrr",
+                            "wrs",
+                            "wru",
+                            "wrv",
+                            "wrw",
+                            "wrx",
+                            "wry",
+                            "wrz",
+                            "wsa",
+                            "wsg",
+                            "wsi",
+                            "wsk",
+                            "wsr",
+                            "wss",
+                            "wsu",
+                            "wsv",
+                            "wtf",
+                            "wth",
+                            "wti",
+                            "wtk",
+                            "wtm",
+                            "wtw",
+                            "wua",
+                            "wub",
+                            "wud",
+                            "wuh",
+                            "wul",
+                            "wum",
+                            "wun",
+                            "wur",
+                            "wut",
+                            "wuu",
+                            "wuv",
+                            "wux",
+                            "wuy",
+                            "wwa",
+                            "wwb",
+                            "wwo",
+                            "wwr",
+                            "www",
+                            "wxa",
+                            "wxw",
+                            "wya",
+
```

Comment:
we should look into updating this list. Also, I don't think we have a way of dealing with multi-licensed items, do we?

Reply:
You're right. We should def make this an array.

For backward-compatibility, I suggest to introduce a definition `#/definitions/license` which can either be of type string, or of type array, similar to the following, which should be valid in a JSON schema file. What do you think?

```json
{
    "anyOf": [
        {
            "type": "string",
        }, 
        {
            "type": "array",
        }
    ]
}
```

Reply:
Also, again, perhaps leave out the enum here, and just leave it as string for the lax schema?

Reply:
I think it's OK to keep it a string, if we use SPDX license expressions. That would also be less ambiguous, with an array the rules for combining elements are implicit, for example do you want to OR the separate licenses or do you want to AND them? I'll make an issue.

Reply:
Okay, keep it as any string for now.

Reply:
I've created issue #128 so the discussion is not blocking merge of this PR